robot/EDA/resources/EDA.py

```diff
@@ -1,12 +1,16 @@
 import logging
 import time
+import datetime
```

Comment:
These imports are unused. Others below on L13 are duplicated. Unless you intended to update these files, they should be reverted to the version on `master`.

Reply:
Yeah i think these changes are due to relEng fix on robot tests. I will remove them. Thanks!